spec/controllers/admin_controller_spec.rb

```diff
@@ -228,12 +292,26 @@ def address_string(address)
 
           result_str_start = csv_header
           result_str_start << member1_info
+          result_str_start << (member1.updated_at.strftime('%F'))
+          result_str_start << ','
```

Comment:
Swap the order of these two lines and the test should pass. (296 should come before 295).

Reply:
there was probably a missing comma but that was not the real problem.